app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -100,6 +100,7 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=12000
 CONFIG_ADSP_IDLE_CLOCK_GATING=y
 CONFIG_ADSP_IMR_CONTEXT_SAVE=n
 CONFIG_PM=y
+CONFIG_POWEROFF=y
```

Comment:
@ranj063 can you add a west patch for CI that pulls in a your zephyr commit. e.g. point west to your PR